chart/templates/triggerer/triggerer-deployment.yaml

```diff
@@ -0,0 +1,212 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Same as in docker compose, I believe this won't work for Airflow pre 2.2. But people will use Helm Chart for 2.1 and 2.0 versions of airflow for quite a while. Should we make some ifs here?

Reply:
The check is there, line 22 (currently).

Swapping $persistence and the version check would probably be a good idea though.

Reply:
Oh yeah. Shame on me :(